src/sentry/tasks/sentry_functions.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+from sentry.tasks.base import instrumented_task
+from sentry.utils import json
+from sentry.utils.cloudfunctions import publish_message
+
+TASK_OPTIONS = {
+    "queue": "app_platform",
+    "default_retry_delay": (60 * 5),  # Five minutes.
+    "max_retries": 3,
+}
+
+
+@instrumented_task("sentry.tasks.sentry_functions.send_sentry_function_webhook", **TASK_OPTIONS)
```

Comment:
You'll want to have `late_acks` on this. Without `late_acks` the MQ messages are acked when the job *starts* and not when the job finishes. This matters because if a deploy happens while you're waiting for a function to be fired the worker will have ACKed the message but not done the work yet.